pkg/virt-config/config-map.go

```diff
@@ -67,56 +62,71 @@ const (
 
 type ConfigModifiedFn func()
 
-func getConfigMap() *k8sv1.ConfigMap {
-	virtClient, err := kubecli.GetKubevirtClient()
-	if err != nil {
-		panic(err)
-	}
-
-	var cfgMap *k8sv1.ConfigMap
-	err = wait.PollImmediate(time.Second*1, time.Second*10, func() (bool, error) {
-
-		namespace, curErr := clientutil.GetNamespace()
-		if err != nil {
-			return false, err
-		}
-
-		cfgMap, curErr = virtClient.CoreV1().ConfigMaps(namespace).Get(ConfigMapName, metav1.GetOptions{})
-
-		if curErr != nil {
-			if errors.IsNotFound(curErr) {
-				logger := log.DefaultLogger()
-				logger.Infof("%s ConfigMap does not exist. Using defaults.", ConfigMapName)
-				cfgMap = &k8sv1.ConfigMap{}
-				return true, nil
-			}
-			return false, curErr
-		}
-
-		return true, nil
-	})
-
-	if err != nil {
-		panic(err)
-	}
-
-	return cfgMap
-}
+// Config has been deprecated in favor of the `KubeVirtSpec.Confguration` option on the
+// KubeVirt Spec. Please add new values there. This is only used for backward
+// compatibility until it can be deprecated
+// type Config struct {
```

Comment:
Were the struct definitions kept intentionally?

Reply:
I did so that the "deprecation" comment was kept. but on second thought it doesn't make much sense to keep it. I have now removed it.